### DISCUSSION TITLE
[Feature] Possibility to turn off reflection and shadow in stage sprites

### DISCUSSION THREAD

**Discussion by MUGEN-Slayer** [ikemen-engine/Ikemen-GO#845] [Feature Request] [answered]
The idea is about being able to turn off reflection/shadow in specific sprites of the stages to have better design results, with parameters like "mask", something like "noshadow" and "noreflection". If if omitted = 0. 
For example, think of the Gen's SFA2 stage.  Possibility of having the sprites in the puddles reflecting normally but the dry part does not. And in the case of the shadow, leave it off in the puddle sprites while on in the dry part of the floor.

Example:

[BG Floor]
type=normal
spriteno=0,0
start=0,0
delta = 1,1
mask=1
**noreflection = 1**

[BG Puddle]
type=normal
spriteno=0,1
start=0,0
delta = 1,1
mask=1
**noshadow = 1**

**Comment by K4thos** [comment, marked as answer]
similar feature suggestion already posted here: https://github.com/Windblade-GR01/Ikemen_GO/issues/133

**Reply by LasombraDemon** [reply]
> similar feature suggestion already posted here: #133

Yes, but this one is more clear. Being able to define Shadow and Reflection "areas" in stages could indeed be useful. There are ways to fake them now, though, via AttachedChar, but that's kind of cheating. xD
